src/app/user-sign-up/register/register.component.scss

```diff
@@ -0,0 +1,68 @@
+/* Import common styling */
+@import "../../layout/variables";
+
+/* Register Form container */
+.form-container {
```

Comment:
The button style will be common to entire project. So whatever common style is there for button, please shift it to global style.

Reply:
IS this change not done? And if no need to shift then please mention reason in the reply here

Reply:
I have created a common button style and tried to import it on the global style but it is not working. For that reason, I have imported them separately.

Reply:
You just need to shift that style to styles.scss file. it should work